kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -110,6 +111,8 @@ func NewIPAMController(cfg config.NodeControllerConfig, c client.Interface, cs k
 
 		// For unit testing purposes.
 		pauseRequestChannel: make(chan pauseRequest),
+
+		datastoreReady: true,
```

Comment:
just a nit, but could you put this up in the same section above (right below `emptyBlocks`)?

The comment implies this might be used for unit testing only but it's not.

Reply:
Ok. Already done.